examples/model_hosting/notebook/simple_scheduled_calculation/SimpleScheduledCalculation.ipynb

```diff
@@ -195,7 +195,7 @@
     "from cognite.model_hosting.schedules import ScheduleOutput\n",
     "\n",
     "df = ScheduleOutput(prediction).get_dataframe(\"mean\")\n",
-    "df.plot(x=\"timestamp\")"
+    "df.plot()"
```

Comment:
Don't need/want `x="timestamp"` anymore?

Reply:
No, the Dataframe returned by DataFetcher now has timestamp as an index instead of a column.